packages/api/src/struct/Auth.ts

```diff
@@ -55,6 +55,7 @@ export class Auth {
 		res.cookie('access_token', credentials.access.token, {
 			path: '/',
 			sameSite: 'strict',
+			expires: new Date(2030, 1),
```

Comment:
In hindsight, just use `credentials.refresh.expiration` - and yes, not `credentials.access`